ADAL/tests/XCTestCase+TestHelperMethods.h

```diff
@@ -40,6 +40,13 @@
 
 #define TEST_SIGNAL dispatch_semaphore_signal(_dsem)
 #define TEST_WAIT dispatch_semaphore_wait(_dsem, DISPATCH_TIME_FOREVER)
+#define TEST_WAIT_NOT_BLOCKING_MAIN_QUEUE \
```

Comment:
@unpluggedk General improvement of async code testing will be done as a part of our UT coverage improvement across the project (including making expectations vs semaphores choice). We can leave it as is as part of this task.